src/components/Calculator.jsx

```diff
@@ -1,31 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Display from './Display';
 import Button from './Button';
 import '../style/Calculator.css';
+import calculate from '../logic/calculate';
 
-const Calculator = () => (
-  <div className="calculator-main-container">
-    <Display />
-    <Button character="AC" />
-    <Button character="+/-" />
-    <Button character="%" />
-    <Button character="÷" otherClass="operator" />
-    <Button character="7" />
-    <Button character="8" />
-    <Button character="9" />
-    <Button character="x" otherClass="operator" />
-    <Button character="4" />
-    <Button character="5" />
-    <Button character="6" />
-    <Button character="-" otherClass="operator" />
-    <Button character="1" />
-    <Button character="2" />
-    <Button character="3" />
-    <Button character="+" otherClass="operator" />
-    <Button character="0" otherClass="zero" />
-    <Button character="·" />
-    <Button character="=" />
-  </div>
-);
+// let obj = {};
+// function handleClick(event) {
+//   obj = calculate(obj, event.target.name);
+//   console.log(obj);
+// }
```

Comment:
- [ ] Please remove commented code as it makes your codebase look cluttered and difficult to read 👍